packages/flutter/test/widgets/semantics_tester.dart

```diff
@@ -232,7 +214,16 @@ class TestSemantics {
   /// The tags of this node.
   final Set<SemanticsTag> tags;
 
-  bool _matches(SemanticsNode node, Map<dynamic, dynamic> matchState, { bool ignoreRect: false, bool ignoreTransform: false, bool ignoreId: false }) {
+  bool _matches(
+    SemanticsNode node,
+    Map<dynamic, dynamic> matchState,
+    {
+      bool ignoreRect: false,
+      bool ignoreTransform: false,
+      bool ignoreId: false,
+      DebugSemanticsDumpOrder childOrder: DebugSemanticsDumpOrder.inverseHitTest,
```

Comment:
Should this be traversalOrder by default?

Reply:
It is now. Thanks for the suggestiong.